CMakeLists.txt

```
@@ -37,6 +37,7 @@ option(ENABLE_SLIC "Build Umpire with SLIC logging" Off)
 option(ENABLE_STATISTICS "Track statistics for allocations and operations" Off)
 option(ENABLE_COVERAGE "Enable code coverage (with GCC)" Off)
 option(ENABLE_TOOLS "Enable Umpire Development Tools" On)
+option(ENABLE_PEDANTIC_WARNINGS "Enable Pedantic Compiler Warnings" On)
```

👍

---

Why is the description in title case? 

```suggestion
option(ENABLE_PEDANTIC_WARNINGS "Enable pedantic compiler warnings" On)
```

---

I don't know.  I just copied from the previous line

---

I think you wrote that line too 😄